openedx/features/caliper_tracking/tests/current/edx.course.enrollment.mode_changed.json

```diff
@@ -0,0 +1,25 @@
+{
+   "username":"honor",
+   "event_source":"server",
+   "name":"edx.course.enrollment.mode_changed",
+   "accept_language":"en-US,en;q=0.9,ur-PK;q=0.8,ur;q=0.7",
+   "time":"2018-10-17T13:08:41.487509+00:00",
+   "agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36",
+   "page":null,
```

Comment:
You've chosen to omit this field in the converted file. Can you please justify your choice?

Reply:
As 'page' is not a required field according to caliper, i think it should be placed in 'extra_fields', i will get it done.